frontend/nginx/nginx.conf

```diff
@@ -10,19 +10,25 @@ events {
 }
 
 http {
-    include       /etc/nginx/mime.types;
     default_type  application/octet-stream;
+    include       /etc/nginx/mime.types;
 
     log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
                       '$status $body_bytes_sent "$http_referer" '
                       '"$http_user_agent" "$http_x_forwarded_for"';
 
     access_log  /var/log/nginx/access.log  main;
-
     sendfile        on;
     #tcp_nopush     on;
-
     keepalive_timeout  65;
     gzip  on;
     include /etc/nginx/conf.d/*.conf;
+
+    server {
+        root /usr/share/nginx/html;
+        index index.html;
+        listen 8000;
+        listen [::]:8000;
+        server_name telemetry.polkadot.io localhost;
```

Comment:
> I tweaked the nginx config in this way so that the "plain" docker version would run in a straight forward way. To me, the nginx config is a detail that should be internal to the container (just as the telemetry backend core and shard serve content, so does the frontend pod, and that is the API that they expose). I'm not sure what the value of moving it out would be?

if you could change the `server_name telemetry.polkadot.io localhost;` to `server_name _;` it would be fine to me as well. If we are not going to change the Nginx configuration frequently, we can copy it in the build time then.

Reply:
As it happens, I just tweaked this line in my latest commit before I saw this :) Since nginx uses this block by default, I just removed the "server_name" line since I believe it's irrelevant (any host header will redirect to this block regardless of what it is)

Reply:
And that sounds good; yup, I don't expect the nginx config to need to change; it's just very basic and generic stuff to serve the static files on the port that we expose in docker :)